libbeat/tests/compose/wrapper.go

```diff
@@ -95,7 +183,66 @@ func (d *wrapperDriver) Up(ctx context.Context, opts UpOptions, service string)
 		args = append(args, service)
 	}
 
-	return d.cmd(ctx, "up", args...).Run()
+	err := d.cmd(ctx, "up", args...).Run()
+	if err != nil {
+		return err
+	}
+	return d.setupAdvertisedHost(ctx, service)
```

Comment:
So this will copy a file with the container host for all containers, but we only need it for Kafka. Is that correct?

Reply:
Yes, this is actually required only by Kafka at the moment.
In python tests I have added an option to the base suite class so it is only copied when required, on Go tests I didn't do it because it would need an extra option to `EnsureUp`.
It doesn't affect builds significatively, but I could give another thought on a good way to add an option for Go tests too.

Reply:
I have added options to `EnsureUp`, so now in Kafka it is done as:
```
        service := compose.EnsureUp(t, "kafka",
                compose.UpWithTimeout(120*time.Second),
                compose.UpWithAdvertisedHostEnvFile,
        )
```